telegram/_inline/inlinequeryresultgif.py

```diff
@@ -77,7 +77,8 @@ class InlineQueryResultGif(InlineQueryResult):
         gif_duration (:obj:`int`): Optional. Duration of the GIF in seconds.
         thumb_url (:obj:`str`): URL of the static (JPEG or GIF) or animated (MPEG4) thumbnail for
             the result.
-        thumb_mime_type (:obj:`str`): Optional. MIME type of the thumbnail.
+        thumb_mime_type (:obj:`str`): Optional. MIME type of the thumbnail, must be one of
+            ``'image/jpeg'``, ``'image/gif'``, or ``'video/mp4'``. Defaults to ``'image/jpeg'``.
```

Comment:
```suggestion
        thumb_mime_type (:obj:`str`): Optional. MIME type of the thumbnail, must be one of
            :mimetype:`image/jpeg`, mimetype:`image/gif`, or mimetype:`video/mp4`. Defaults to mimetype:`image/jpeg`.
```

Reply:
It seems logical, but in docs it renders as italics, and we want to signal to the user that they should put literals in there. 

Maybe I should use
```
``'image/jpeg'``
```
in the args and 
```
:mimetype:`image/jpeg`
```
in the attrs. 

Or I'm being too meticulous :)

Reply:
The sphinx-docs list the `:mimetype:` role in the section "Other semantic markup - The following roles don’t do anything special except formatting the text in a different style:" (see [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-mimetype)).

Having the text render as monospaced literals makes sense and that's more important to me than to use the :`mimetype` role just for the sake of it …

Reply:
OK! No changes here, then.